tox.ini

```diff
@@ -1,7 +1,8 @@
 [tox]
 downloadcache = {toxworkdir}/cache/
 
-envlist = py2.7-django{1.11,1.10,1.9,1.8}
+envlist = py{27,36,37}-django{1.11,1.10,1.9,1.8}
```

Comment:
By convention we don't usually have the dots in the django versions.  You should make this `py{27,36,37}-django{111}` since we don't need to test on those older versions anymore either.  You should update the deps below as well to match this convention.